docs/event/assets.md

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_position: 4
+---
+
+# Assets
+
+Assets are files associated with an event.
+Each asset has some asset-metadata that is stored in the DB, while the file itself is stored in the file system and is referenced by the database.
+
+Every asset has this metadata attached to it:
+- `id: ID`: unique identifier among all assets of all events. Assigned by Opencast and unchangable.
+- `flavor: NonBlankAsciiString` <sup>(1?)</sup>
```

Comment:
This should be its own datatype. `flavor` has a specific syntax. I guess `[0-9a-Z+-]/[0-9a-Z+-]` is fine, but maybe some additional characters are used.